### PR TITLE
[refactor] update stop_be.sh to avoid error message

### PR DESCRIPTION
**Pull request by englefly**
# Proposed changes
execute bin/stop_be.sh without any parameter, we will receive a error message:
`./stop_be.sh: 第 26 行:[: =: 期待一元表达式`
This pr aims to remove this unnecessary message.

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by adonis0147** [comment]
Duplicated, see #10689 .

File: bin/stop_be.sh [hunk 1]
```diff
@@ -23,7 +23,7 @@ export DORIS_HOME=`cd "$curdir/.."; pwd`
 export PID_DIR=`cd "$curdir"; pwd`
 
 signum=9
-if [ $1 = "--grace" ]; then
+if [ $# -gt 0 ] && [ $1 = "--grace" ]; then
```
Comment (adonis0147) [inline bin/stop_be.sh:26]:
```suggestion
if [[ "$1" == "--grace" ]]; then
```

**Review by adonis0147 — CHANGES_REQUESTED** [review]

**Review by adonis0147 — DISMISSED** [review]
LGTM

**Review by yiguolei — DISMISSED** [review]
LGTM

**Review by yiguolei — APPROVED** [review]
lgtm

**Pull request merged**